lib/components/narrative/narrative-itineraries-header.tsx

```diff
@@ -206,7 +206,7 @@ export default function NarrativeItinerariesHeader({
                 )}
               </StyledIconWrapper>
             </button>
-            <SortResultsDropdown
+            <Dropdown
               id="sort-results"
               label={sortResultsLabel}
               name={sortText}
```

Comment:
Pretty sure there's no `name` prop on the new `Dropdown` so this is causing an error!